third_party/2and3/emoji/unicode_codes.pyi

```diff
@@ -0,0 +1,6 @@
+from typing import Dict, Text
+
+EMOJI_UNICODE: Dict[Text, Text]
+EMOJI_UNICODE_ALIAS: Dict[Text, Text]
```

Comment:
This is actually called `EMOJI_ALIAS_UNICODE`. (Must be fixed in the re-export above as well.)